File: HW1_python/6.py
a = int(input())
b = int(input())
c = int(input())
D = (b ** 2) * (-4 * a * c)
if D >= 1:
    x1 = (-b + (D**0.5)) / (2 * a)
    x2 = (-b - (D**0.5)) / (2 * a)
    print(int(x1), int(x2))
elif D == 0:
    x = (-b / (2 * a))
    print(int(x))

File: HW5_C/6.c
#include <stdio.h>

void input(int arr[], int n) {
    for(int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
}

void output(int arr[], int n) {
    for(int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
}


int abs_Array(int arr[], int n) {
    int *cout = arr, out_func = 0;
    for(int i = 0; i < n; i++) {
        if (*cout < 0) {
            *cout *= -1;
            out_func++;
        }
        cout++;
    }
    return out_func;
}

int main() {
    int n; scanf("%d", &n);
    int arr[n];
    input(arr, n);
    printf("%d ", abs_Array(arr, n));
    output(arr, n);
}